src/drag-and-drop/components/app.tsx

```diff
@@ -66,10 +66,38 @@ export const baseAuthoringProps = {
               title: "Image URL",
               type: "string",
               format: "uri"
+            },
+            imageWidth: {
+              title: "Image width",
+              type: "number"
+            },
+            imageHeight: {
+              title:"Image height",
+              type: "number"
+            },
```

Comment:
Why do we need now image width and height? It worked without it. I think it might be annoying for authors to provide pixel perfect dimensions for each image they use.

Reply:
It's an optionaly porperty if somebody needed to adjust their image to fit in the container. I ran into that problem when I was trying to find a water droplet to test with.